src/atomate2/aims/flows/phonons.py

```diff
@@ -138,3 +138,15 @@ def __post_init__(self):
                 self.phonon_displacement_maker = PhononDisplacementMakerSocket()
             else:
                 self.phonon_displacement_maker = PhononDisplacementMaker()
+
+    @property
```

Comment:
Should we add `@abstractmethod` here?

Reply:
But why should it be still abstract there? As far as I understand the decorator, it indicates that inheriting classes have to implement the method.